AppCenter/AppCenter/Internals/Sender/MSHttpSender.h

```diff
@@ -34,10 +34,14 @@ NS_ASSUME_NONNULL_BEGIN
 /**
  *  Send data to backend
  * @param data A data instance that will be transformed request body.
+ * @param appSecret The appSecret.
  * @param callId A unique ID that identify a request.
  * @param handler Completion handler
  */
-- (void)sendAsync:(NSObject *)data callId:(NSString *)callId completionHandler:(MSSendAsyncCompletionHandler)handler;
+- (void)sendAsync:(NSObject *)data
+            appSecret:(NSString *)appSecret
```

Comment:
nit - `:` not aligned

Reply:
same as above

Reply:
Our code formatter does that :slightly_smiling_face: